src/api/interactivity.js

```diff
@@ -3,6 +3,66 @@ import Layer from './layer';
 import { WM_R, projectToWebMercator } from './util';
 import { wToR } from '../client/rsys';
 
+
+/**
+ * 
+ * FeatureEvent objects are fired by {@link carto.Interactivity|Interactivity} objects.
```

Comment:
what is a feature event? what is it used for?

Reply:
It is an `Object`, usage is end-app scope, not ours.